### ISSUE TITLE
Gradle build fails

### ISSUE THREAD

**Issue by askask** [eilslabs/BatchEuphoria#6]
When trying to run `gradle build`, it fails with the following output:
```
FAILURE: Build failed with an exception.

* Where:
Build file '/data/code/BatchEuphoria/build.gradle' line: 21

* What went wrong:
A problem occurred evaluating root project 'BatchEuphoria'.
> Command failed with code 128: '/usr/bin/env git -C /data/code/BatchEuphoria describe --tags --dirty'
  Error output: 'fatal: No names found, cannot describe anything.'
```

**Comment by dankwart-de** [comment]
Works for me:

```
✔ ~/Projekte/RoddyInfra/BatchEuphoria [askaskReview L|✚ 8…1⚑ 1] 
12:46 $ gradle -Dhttp.proxyHost=www-int2.inet.dkfz-heidelberg.de -Dhttp.proxyPort=80 -Dhttps.proxyHost=www-int2.inet.dkfz-heidelberg.de -Dhttps.proxyPort=80 build
:compileJava UP-TO-DATE
:compileGroovy
Note: /data/michael/Projekte/RoddyInfra/BatchEuphoria/src/main/groovy/de/dkfz/roddy/execution/jobs/FakeJobID.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
:processResources UP-TO-DATE
:classes
:jar
:startScripts
:distTar
:distZip
:assemble
:compileTestJava UP-TO-DATE
:compileTestGroovy
:processTestResources UP-TO-DATE
:testClasses
:test
:check
:build

BUILD SUCCESSFUL

```

**Comment by askask** [comment]
It looks like Travis has the same problem: https://travis-ci.org/eilslabs/BatchEuphoria/builds/239918962

Could you clone the repository to an empty folder and try it again?

**Comment by dankwart-de** [comment]
You're right, I can remember the problem. I could solve it locally by adding a tag like e.g. 0.0.01 or so. I tried to repeat it on GitHub, but it didn't work with Travis. I need to talk to vinjana to solve it.

**Comment by dankwart-de** [comment]
Fixed by pushing a random tag 0.0.01
